sources/platform/actors/development/actor_definition/source_code/container_web_server.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Container web server
 description: Learn about how to run a web server inside your Actor, which enables you to communicate with the outer world via both UI and API.
-slug: /actors/development/container-web-server
+slug: /actors/development/actor-definition/source-code/container-web-server
```

Comment:
THe container web server really feels weird under source code. It does not have a lot to do with the actor's source code, it's mostly about a platform feature that exposes a port and then its a tutorial how to use it.